session/manager.go

```diff
@@ -26,47 +26,41 @@ import (
 // ServiceConfigProvider defines configuration providing dependency
 type ServiceConfigProvider func() (ServiceConfiguration, error)
 
+// IDGenerator defines method for session id generation
+type IDGenerator func() SessionID
```

Comment:
Can we assume, that future implementations will always succeed and never posibliy return error? I.e. true crypto random generator? Maybe it's worth to define error as part of result - it's more relaxed definition.

Reply:
I just remove error from `SessionMap` just because current implementation `storageMemory` dont return error (because of feadback).
So.. it look that dont do assumtions on future, w;ll implement when needed.